src/PatternPluginManager.php

```diff
@@ -0,0 +1,71 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * @see       https://github.com/laminas/laminas-cache for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-cache/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-cache/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Laminas\Cache;
+
+use Laminas\ServiceManager\AbstractPluginManager;
+use Laminas\ServiceManager\Factory\InvokableFactory;
+use Webmozart\Assert\Assert;
+
+class PatternPluginManager extends AbstractPluginManager
+{
+    /**
+     * @var bool
+     */
+    protected $sharedByDefault = false;
+
+    protected $aliases = [
+        'callback' => Pattern\CallbackCache::class,
+        'Callback' => Pattern\CallbackCache::class,
+        'capture' => Pattern\CaptureCache::class,
+        'Capture' => Pattern\CaptureCache::class,
+        'class' => Pattern\ClassCache::class,
+        'Class' => Pattern\ClassCache::class,
+        'object' => Pattern\ObjectCache::class,
+        'Object' => Pattern\ObjectCache::class,
+        'output' => Pattern\OutputCache::class,
+        'Output' => Pattern\OutputCache::class,
+
+        // Legacy Zend Framework aliases
+        \Zend\Cache\Pattern\CallbackCache::class => Pattern\CallbackCache::class,
+        \Zend\Cache\Pattern\CaptureCache::class => Pattern\CaptureCache::class,
+        \Zend\Cache\Pattern\ClassCache::class => Pattern\ClassCache::class,
+        \Zend\Cache\Pattern\ObjectCache::class => Pattern\ObjectCache::class,
+        \Zend\Cache\Pattern\OutputCache::class => Pattern\OutputCache::class,
```

Comment:
Get rid of these; you're removing aliases like these from other plugin managers, and you've removed the bridge package, because it targets a new major version. Time to get rid of the legacy!

Reply:
https://github.com/laminas/laminas-cache/pull/55